### PR TITLE
requirements.txt v0.2 closes #81

### LINKED ISSUE #81: Create requirements.txt
We should develop a [requirements.txt](https://devcenter.heroku.com/articles/python-pip) file that installs the dependencies for our application.

This requirements file will be utilized for continuous integration testing, and simplifying distribution.

It should supplant a decent chunk of Software/setup.sh

### PR DESCRIPTION
**Pull request by gmross**
Attempted to create a requirements.txt file.  From what I could read, it consists of the arguments in the commands for setup.sh (the package names).  Not entirely sure if any of it is right

### CODE REVIEW

**Comment by ophilli** [comment]
@gmross Looks awesome man! Great job. I've got a few comments though - Please review and implement [this.](https://github.com/waffleio/waffle.io/wiki/FAQs#prs-connect-keywords)

Additionally, please review the best practices for requirements.txt [here.](https://devcenter.heroku.com/articles/python-pip)

File: Software/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+setuptools
+pip
+python-dev
+linux-headers
+python3>=3.4,<3.5
```
Comment (ophilli) [inline Software/requirements.txt]: None of the packages above here belong in requirements.txt.

You're writing a [pip requirements file](https://pip.pypa.io/en/stable/user_guide/#requirements-files), and pip is the package manager for python. That means that the only things we can specify are things that pip can install for us.

Finally, please review the best practices [here](https://devcenter.heroku.com/articles/python-pip) and implement them below.

**Review by ophilli — CHANGES_REQUESTED** [review]

**Comment by ophilli** [comment]
Great job @gmross! I'm going to let somebody else finish this review. 

Looks like the link I gave you on connection words wasn't fully complete - try using a github [closing](https://help.github.com/articles/closing-issues-via-commit-messages/) keyword. This keyword will close the issue when we merge this PR.

**Comment by r3cebarnett** [comment]
Looks good to me! If I missed something it will turn up very soon anyways.

**Review by r3cebarnett — APPROVED** [review]

**Review by ophilli — APPROVED** [review]

**Pull request merged**